pyproject.toml

```
@@ -1,72 +1,68 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
 [project]
 name = "mne-qt-browser"
 description = "A new backend based on pyqtgraph for the 2D-Data-Browser in MNE-Python"
 readme = "README.md"
-requires-python = ">=3.8"
-license = {file = "LICENSE"}
+requires-python = ">=3.9"
+license = { file = "LICENSE" }
 keywords = ["science", "neuroscience", "psychology"]
-authors = [
-  {name = "Martin Schulz", email = "dev@earthman-music.de"},
-]
+authors = [{ name = "Martin Schulz", email = "dev@earthman-music.de" }]
```

```suggestion
authors = [{ name = "Martin Schulz", email = "dev@mgschulz.de" }]
```
I have a new e-mail-adress for software-development. Would you kindly change that for me?

---

Thank you @hoechenberger